libraries/GCS_MAVLink/GCS.h

```diff
@@ -28,6 +28,7 @@
 #include <AP_AHRS/AP_AHRS_config.h>
 #include <AP_Arming/AP_Arming_config.h>
 #include <AP_Airspeed/AP_Airspeed_config.h>
+#include <AP_Scheduler/AP_Scheduler.h>
```

Comment:
Move the function implementation to the .cpp file.

It's too big to go in the header.

Reply:
Moved to cpp